src/components/tx/modals/NewTxModal/ReplacementModal.tsx

```diff
@@ -0,0 +1,140 @@
+
+import css from './styles.module.css'
+
+const wrapIcon = (icon: React.ReactNode) => <div className={css.circle}>{icon}</div>
+
+const steps = [
+  {
+    label: 'Create new transaction with same nonce',
+    icon: <div className={css.redCircle} />,
+  },
+  {
+    label: 'Collect confirmations from owners',
+    icon: wrapIcon(<CheckIcon fontSize="small" color="border" />),
+  },
+  {
+    label: 'Execute replacement transaction',
+    icon: wrapIcon(<SvgIcon component={RocketIcon} inheritViewBox fontSize="small" color="border" />),
+  },
+  {
+    label: 'Initial transaction is replaced',
+    icon: wrapIcon(<SvgIcon component={DeleteIcon} inheritViewBox fontSize="small" color="border" />),
+  },
+]
+
+const btnWidth = {
+  width: {
+    xs: 240,
+    sm: '100%',
+  },
+}
+
+const ReplacementModal = ({
+  txNonce,
+  onClose,
+  onTokenModalOpen,
+  onNFTModalOpen,
+  onRejectModalOpen,
+}: {
+  txNonce: number
+  onClose: () => void
+  onTokenModalOpen: () => void
+  onNFTModalOpen: () => void
+  onRejectModalOpen: () => void
+}) => {
+  return (
+    <ModalDialog open dialogTitle={`Replace transaction with nonce ${txNonce}`} onClose={onClose}>
+      <DialogContent className={css.container}>
+        <Typography variant="h5" mb={1} textAlign="center">
+          Need to replace or discard this transaction?
+        </Typography>
+        <Typography variant="body1" textAlign="center">
+          A signed transaction cannot be removed but it can be replaced with a new transaction with the same nonce.
+        </Typography>
+        <Stepper alternativeLabel className={css.stepper}>
+          {steps.map(({ label }) => (
+            <Step key={label}>
+              <StepLabel StepIconComponent={({ icon }) => steps[Number(icon) - 1].icon}>
+                <Typography variant="body1" fontWeight={700}>
+                  {label}
+                </Typography>
+              </StepLabel>
+            </Step>
+          ))}
+        </Stepper>
+      </DialogContent>
+      <DialogActions className={css.container}>
+        <Grid
+          container
+          alignItems="center"
+          justifyContent="center"
+          flexDirection={{
+            xs: 'column',
+            sm: 'row',
+          }}
+        >
+          <Grid item xs={12}>
+            <Typography variant="body2" textAlign="center" fontWeight={700} mb={3}>
+              Select the type of replacement transaction
+            </Typography>
+          </Grid>
+          <Grid item container justifyContent="center" gap={1} xs={12} sm>
+            <SendTokensButton onClick={onTokenModalOpen} sx={btnWidth} />
+            <SendNFTsButton onClick={onNFTModalOpen} sx={btnWidth} />
+          </Grid>
+          <Grid item>
+            <Typography variant="body2" className={css.or}>
+              or
+            </Typography>
+          </Grid>
+          <Grid item container xs={12} sm sx={{ justifyContent: { xs: 'center', sm: 'flex-start' } }}>
```

Comment:
On some widths below 600px the Button and text are not collapsing. We could explicitly set a column based layout in viewports < 600px to fix this.
<img width="503" alt="Screenshot 2022-11-15 at 14 58 23" src="https://user-images.githubusercontent.com/5880855/201937815-79de779d-54f0-4504-a8c3-22a48d13e34d.png">

Reply:
This should now work.